app/build.gradle.kts

```diff
@@ -7,6 +7,10 @@ android {
     namespace = "com.mstudio.superheroarch"
     compileSdk = 34
 
+    viewBinding{
```

Comment:
you are not using this.

Reply:
True, it was the first way I used to do step 5, but I did not eliminate it when I stopped using it.